djvm/osgi/test.bndrun

```diff
@@ -0,0 +1,51 @@
+# Bndrun for OSGi DJVM testing
+
+-tester: biz.aQute.tester.junit-platform
+
+-runee: JavaSE-1.8
+-runfw: org.apache.felix.framework
+-resolve.effective: active
+-runpath: \
+    org.jetbrains.kotlin.osgi-bundle;version='${project.kotlin_version}',\
+    ${project.kotlinJars}
+-runsystempackages: kotlin.jdk7, kotlin.streams.jdk8
+-runproperties: \
+    org.osgi.framework.bootdelegation=sun.reflect,\
+    org.osgi.service.log.admin.loglevel=DEBUG,\
```

Comment:
Do we want this loglevel to be `DEBUG`?

Reply:
This is the ROOT logging level for the OSGi container, but the `logback.xml` file sets the logging threshold to `INFO`so we don't see any `DEBUG` messages.